projects/storefrontlib/cms-structure/page/page-layout/page-layout.service.ts

```diff
@@ -76,32 +81,37 @@ export class PageLayoutService implements OnDestroy {
       map((breakpoint) => {
         if (!this.config.layoutSlots) {
           // no layout config available
-          return null;
+          return undefined;
         }
         const pageTemplateConfig = this.config.layoutSlots[pageTemplate];
         const config = this.getResponsiveSlotConfig(
           <LayoutSlotConfig>pageTemplateConfig,
           'pageFold',
           breakpoint
         );
-        return config ? config.pageFold : null;
-      })
+        return config ? config.pageFold : undefined;
+      }),
+      filter(isNotUndefined)
```

Comment:
This function used to emit null when no config was returned.
Now it doesn't emit anything.  Could this be an issue for callers of this function?